### DISCUSSION TITLE
What does it mean when the log prints “Running install script”

### DISCUSSION THREAD

**Discussion by includeios** [pnpm/pnpm#4451] [Q&A]
My project is very slow when installing `ttf2woff2`,This makes the installation time of the whole project even longer than npm.
I don't know what's going on at this step, can anyone help to answer this question and How can I make changes to make the project install faster, thanks。
my environment
node: v16.13.2
pnpm: 6.32.3
<img width="897" alt="image" src="https://user-images.githubusercontent.com/18004081/159123821-21498c3e-0bed-4d3c-aa45-83d9f55e144b.png">

**Comment by Jack-Works** [comment]
https://github.com/nfroidure/ttf2woff2/blob/master/

It's compiling a native C++ Node module. I guess there is nothing you can do to speed up this.

**Reply by zkochan** [reply]
They can turn on side-effects-cache (https://pnpm.io/npmrc#side-effects-cache)

It will be `true` by default from pnpm v7

**Reply by includeios** [reply]
thanks! i will try it.

**Reply by includeios** [reply]
Another question is why does pnpm execute this compilation when installing?Is it for a faster project launch?

**Reply by zkochan** [reply]
It is a postinstall script of a dependency. pnpm cannot control it.
